07-data_pipelines.Rmd

```diff
@@ -157,3 +157,74 @@ Link: https://airflow.tools.alpha.mojanalytics.xyz
 You can manually start ("trigger") the pipeline with this button:
 
 ![](images/pipelines/trigger.png)
+
+
+## Test pipeline in your own Airflow sandbox
+
+You can deploy your own sandboxed version of Airflow to test Dags and docker images on.
+
+### Start Airflow sandbox
+
+Click the `Deploy` button next to `airflow-sqlite` in the Analytical tools section of the control panel. This should take a few minutes to deploy. Then Click the `Open` button to open your Airflow sandbox in a new window.
+
+This will creat an `airflow` directory in your home directory on the Analytical Platform. In this airflow directory there are three more directories `db`, `dags` and `logs`. The `dags` directory is where you will store your test dag files.
+
+### Create a test Dag
+
+Using Jupiter lab on the Analytical Platform create a python file in the `airflow/dags` directory in your home directory in the Analytical Platform (eg. `test_dag.py`). Airflow will automatically scan this directory for dag files every three minutes.
+
+See above for creating a Docker image to run the dag if you are using `KubernetesPodOperator`. Concourse will automatically build images of PR's to github airflow repos and will be tagged with the name of the PR.
+
+Example Dag file contents:
+
+
+    from airflow.utils.dates import days_ago
+    from airflow.utils.log.logging_mixin import LoggingMixin
+    from airflow.models import DAG
+
+    log = LoggingMixin().log
+
+    try:
```

Comment:
Do you need to import `KubernetesPodOperator` in a `try` block? In theory it should be there so we can simplify the example (unless I'm missing something of course)